server/gc/src/test/java/org/apache/accumulo/gc/SimpleGarbageCollectorTest.java

```diff
@@ -137,6 +137,21 @@ public void testMoveToTrash_NotUsingTrash() throws Exception {
     assertFalse(gc.moveToTrash(path));
   }
 
+  @Test
+  public void testMoveToTrash_NotUsingTrash_importsOnlyEnabled() throws Exception {
+    systemConfig.set(Property.GC_TRASH_IGNORE.getKey(), "true");
+    systemConfig.set(Property.GC_TRASH_IGNORE_IMPORTS_ONLY.getKey(), "true");
+    Path iFilePath = new Path("Ifile");
```

Comment:
Please take a look at TabletFileTest.testValidPaths for some more realistic file and path names.

Reply:
Updated the PR with paths similar to those in testValidPaths